### ISSUE TITLE
Why is forms::validID() requiring an objectID in the query string?

### ISSUE THREAD

**Issue by MichaelRBond** [wvulibraries/mfcs#191]
This prevents us from using it to validate formIDs in some circumstances.

**Comment by ddavisgraphics** [comment]
I think this is legacy script, but I don't know what it would break if it is changed.

**Comment by MichaelRBond** [comment]
Right. because we don't have unit testing in place for this project ;-). 

I have feeling that it stems from laziness. However, it may no longer be needed with the new validation that is in place. we'll need to investigate.
